backend/firestore.rules

```diff
@@ -6,4 +6,10 @@ service cloud.firestore {
       allow read, write: if request.auth.uid == userId
     }
   }
+
+  match /databases/{database}/documents {
+    match /scenarios/{documents=**} {
+      allow read, write: if request.auth.uid != null;
+    }
+  }
```

Comment:
At the moment, anyone who's authenticated can access the scenarios.  This should be tightened up for sure, but probably shouldn't be a launch blocker.

Reply:
Let's just make sure to file issues for anything we know had to be done but defer until after the next release.